src/main/java/org/openpnp/gui/JobPlacementsPanel.java

```diff
@@ -72,6 +74,10 @@
     private static Color statusColorWarning = new Color(252, 255, 157);
     private static Color statusColorReady = new Color(157, 255, 168);
     private static Color statusColorError = new Color(255, 157, 157);
+    private static Color cellColorSelected = new Color(51, 153, 255);
```

Comment:
This color ends up clashing with the system color for a selected row. Instead, use UIManager to get the correct Swing color. See http://alvinalexander.com/java/java-uimanager-color-keys-list

Example:
![screen shot 2017-03-29 at 9 38 39 am](https://cloud.githubusercontent.com/assets/1182323/24460303/876d3d22-1463-11e7-971b-82f3ab661542.png)